### ISSUE TITLE
Performance!

### ISSUE THREAD

**Issue by ircube** [tejitak/left-swipe-action#5]
Hi, i'm using this great component, but I'm having some problems:

I'm importing left-swipe-action and a paper-fab.
when the paper-fab button is used.
(the left-swipe-action is not used yet)
on each click to the paper-fab element, the animation of paper-ripple is slowing.

... but if the import of left-swipe-action is deleted, I can click on the paper-fab button any times, without noticing the animation of paper-ripple slows.

i'm presuming the left-swipe-action calling an animation function on each click, and it is stacking on the browser's memory.
the code:

``` html
<!doctype html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>infra</title>
  <script src="components/webcomponentsjs/webcomponents-lite.js"></script>

  <link rel="import" href="components/paper-styles/paper-styles.html">

  <link rel="import" href="components/left-swipe-action/left-swipe-action.html">

  <link rel="import" href="components/paper-fab/paper-fab.html">
  <link rel="import" href="components/iron-icon/iron-icon.html">
  <link rel="import" href="components/iron-icons/iron-icons.html">


</head>
<body class="fullbleed">
<paper-fab icon="delete" ></paper-fab>
</body>
</html>
```

a capture of the browser's Timeline:

![image](https://cloud.githubusercontent.com/assets/913069/8244896/f161a002-15f4-11e5-81d3-03b3b58036fd.png)

**Comment by tejitak** [comment]
Hi @ircube,

Thank you for your report. I noticed that the cause is by "polymer-gestures" which left-swipe-action includes internally. 

https://github.com/Polymer/polymer-gestures/

I removed the import in left-swipe-action.html, the issue does not occur.
The gestures library is official plugin but it was for polymer 0.4 or 0.5. It obviously does not look currently supported & maintained for 1.0.

I'll consider the alternative way...!
Firstly, Thanks for the report!
